cypress/integration/route_spec.js

```diff
@@ -0,0 +1,98 @@
+describe('Sea route tests', function () {
+
+    before (function () {
+	cy.login();
+    });
+
+    beforeEach(function () {
+        // Session cookies will not be cleared before the NEXT test starts
+        cy.preserveSessionOnce();
+    });
+
+
+    it ("Create new route", function () {
+	cy.visit ("/ote/#/routes/");
+
+	cy.wait(500);
```

Comment:
Is it necessary to use wait? It will make our tests more flaky. Best practices regarding to waiting: https://docs.cypress.io/guides/references/best-practices.html#Unnecessary-Waiting

Reply:
Wait removed